src/Popover.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import system from 'styled-system'
+import theme from './theme'
+import Box from './Box'
+
+const rootPosition = props =>
+  props.open
+    ? {
+        position: 'fixed',
+        zIndex: system.theme('zIndex.2')(props),
+        top: props.top || 0,
+        display: 'flex',
+        flexDirection: 'column',
+        backgroundColor: system.theme('colors.' + props.bg, 'white'),
+
+        [system.theme('mediaQueries.1')(props)]: {
+          position: 'relative',
+          top: 'auto',
+          display: 'block'
+        }
+      }
+    : null
+
+const Root = styled(Box)`
+  right: 0;
+  bottom: 0;
+  left: 0;
+  ${rootPosition};
+`
+
+Root.defaultProps = {
+  theme
+}
+
+class Popover extends React.Component {
+  static propTypes = {
+    open: PropTypes.bool.isRequired,
+    onDismiss: PropTypes.func.isRequired
+  }
+
+  constructor() {
```

Comment:
Do we really need the constructor? Seems like we can just have handleDismiss as a class function?

Reply:
I prefer custom class methods in the constructor to avoid the `.bind` syntax, but I don't mind either way